### PR TITLE
pages to view, manage and create projects/services

### PR DESCRIPTION
**Pull request by amitojsingh366**
Screenshot 2022-12-08 at 8 09 46 PM
Screenshot 2022-12-08 at 8 10 09 PM
Screenshot 2022-12-08 at 8 14 42 PM
Screenshot 2022-12-10 at 1 36 03 AM
Screenshot 2022-12-10 at 3 07 39 PM
Screenshot 2022-12-10 at 1 36 53 AM

### CODE REVIEW

**Comment by TheOtterlord** [comment]
This looks sick. I do have one major thing to note however. The user won't be creating services using the current endpoint unless their administrator (I forgot to mention this sry). so the current service creation page should probably be restricted to admins or rewritten for the future, as once we define our first service, we'll probably need a general service form page that can fill in with required fields. It might still be worth keeping the current one though, as a hard coded page just for admins.

thoughts? @HoloPanio

**Comment by HoloPanio** [comment]
I already informed him of the admin only condition of the route, but it would've had to been changed regardless.

As far as quality of life, @amitojsingh366 can you make the params field a bit bigger for readability's sake?

**Comment by HoloPanio** [comment]
Also this looks amazing, I love it a lot

**Comment by amitojsingh366** [comment]
> As far as quality of life, @amitojsingh366 can you make the params field a bit bigger for readability's sake?

@HoloPanio you want the params field on the create service page to be bigger, correct?

**Comment by amitojsingh366** [comment]
@HoloPanio I am now using a text area for the params field on create service page, this allows the user to resize it as they wish. screenshot updated.

File: dashboard/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@storybook/addon-postcss": "^2.0.0",
     "axios": "^0.24.0",
     "formik": "^2.2.9",
+    "moment": "^2.29.4",
```
Comment (goldyydev) [inline dashboard/package.json]: [Do we really need momentjs?](https://you-dont-need.github.io/You-Dont-Need-Momentjs/)

File: dashboard/src/modules/services/ViewServicePage.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                >
+                  {serviceData?.id}
+                </p>
+              </div>
+
+            </div>
+            <div className="flex flex-col gap-4 p-4 w-full">
+              <p>Name: {serviceData?.name}</p>
+              <p>ID: {serviceData?.id}</p>
+              <p>Created: {moment(serviceData?.createdAt).format('MMMM Do YYYY, h:mm:ss a')}</p>
```
Comment (goldyydev) [inline dashboard/src/modules/services/ViewServicePage.tsx]: Might be worth it to use Intl.DateTimeFormat instead.

**Review by goldyydev — CHANGES_REQUESTED** [review]

File: dashboard/src/modules/services/ViewServicePage.tsx (hunk 2 above)
Comment (goldyydev) [reply dashboard/src/modules/services/ViewServicePage.tsx]: Or dayjs

**Review by goldyydev — APPROVED** [review]

**Review by TheOtterlord — APPROVED** [review]

**Pull request merged**